ompi/mpi/fortran/use-mpi-f08/mod/mpi-f08-rename.h

```diff
@@ -401,16 +401,6 @@
 #define MPI_Improbe PMPI_Improbe
 #define MPI_Imrecv_f08 PMPI_Imrecv_f08
 #define MPI_Imrecv PMPI_Imrecv
-#define MPI_Ineighbor_allgather_f08 PMPI_Ineighbor_allgather_f08
-#define MPI_Ineighbor_allgather PMPI_Ineighbor_allgather
-#define MPI_Ineighbor_allgatherv_f08 PMPI_Ineighbor_allgatherv_f08
-#define MPI_Ineighbor_allgatherv PMPI_Ineighbor_allgatherv
-#define MPI_Ineighbor_alltoall_f08 PMPI_Ineighbor_alltoall_f08
-#define MPI_Ineighbor_alltoall PMPI_Ineighbor_alltoall
-#define MPI_Ineighbor_alltoallv_f08 PMPI_Ineighbor_alltoallv_init_f08
-#define MPI_Ineighbor_alltoallv PMPI_Ineighbor_alltoallv
-#define MPI_Ineighbor_alltoallw_f08 PMPI_Ineighbor_alltoallw_f08
-#define MPI_Ineighbor_alltoallw PMPI_Ineighbor_alltoallw
```

Comment:
Why were these removed?

Reply:
i thought one of the compilers was complaining about multiple defines - which is a good thing.  what I don't understand is why i didn't see the same for other entries.  it may be we can get rid of much of the mpi-f08-rename.h content since its now being generated in the python code which generates much of the content for the mpi_f08 source code.

Reply:
line 410 in the file was incorrect